ext/cl_khr_pci_bus_info.asciidoc

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018-2021 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[[cl_khr_pci_bus_info]]
+== PCI Bus Information Query
+
+This extension adds a new query to obtain PCI bus information about an OpenCL
+device.
+
+Not all OpenCL devices have PCI bus information, either due to the device not
+being connected to the system through a PCI interface or due to platform
+specific restrictions and policies. Thus this extension is only expected to be
+supported by OpenCL devices which can provide the information.
+
+As a consequence, applications should always check for the presence of the
+extension string for each individual OpenCL device for which they intend to
+issue the new query for and should not have any assumptions about the
+availability of the extension on any given platform.
+
+=== General information
+
+==== Name Strings
+
+`cl_khr_pci_bus_info`
+
+==== Version History
+
+[cols="1,1,3",options="header",]
+|====
+| *Date*     | *Version* | *Description*
+| 2021-04-19 | 1.0.0     | Initial version.
+|====
+
+==== Dependencies
+
+This extension is written against the OpenCL API Specification Version V3.0.6.
+
+This extension requires OpenCL 1.0.
+
+=== New API Types
+
+Structure returned by the device info query for `CL_DEVICE_PCI_BUS_INFO_KHR`:
+
+[source,c]
+----
+typedef struct _cl_device_pci_bus_info_khr {
+    cl_uint   pci_domain;
+    cl_uint   pci_bus;
+    cl_uint   pci_device;
+    cl_uint   pci_function;
+} cl_device_pci_bus_info_khr;
+----
+
+=== New API Enums
+
+Accepted value for the _param_name_ parameter to *clGetDeviceInfo*:
+
+[source,c]
+----
+#define CL_DEVICE_PCI_BUS_INFO_KHR  0x410F
+----
+
+=== Modifications to the OpenCL API Specification
+
+==== Section 4.2 - Querying Devices:
+
+Add to Table 5 - OpenCL Device Queries:
+
+[caption="Table 5. "]
+.OpenCL Device Queries
+[width="100%",cols="<30%,<20%,<50%",options="header"]
+|====
+| *cl_device_info* | Return Type | Description
+| `CL_DEVICE_PCI_BUS_INFO_KHR`
+  | `cl_device_pci_bus_info_khr`
+    | Returns PCI bus information for the device.
+
+      The PCI bus information is returned as a single structure that includes
+      the PCI bus domain, the PCI bus identifier, the PCI device identifier, and
+      the PCI device function identifier.
+
+|====
+
+=== Issues
+
+None.
+
+//. Title
```

Comment:
I'm guessing the lines below should be removed.

Reply:
Fixed - I ended up removing the entire "Issues" section since it was empty.